eloquentjs/2_1_looping_a_triangle.js

```diff
@@ -0,0 +1,12 @@
+function DrawTriangle(height){ // Draws a right triangle
+    let tri = '';
+    for(let i =0; i <= height; i++){
+        for(let j =0; j< i; j++){
+            tri+="#";
+        }
+        tri+="\n";
```

Comment:
This code is totally fine, but since you mentioned mitigating calls to the console it's probably worth knowing that concatenating into a string in js (as in most languages) actually allocates a brand new string, copies the old string into it, and then copies the new stuff in. Calling the console would likely be about the same at this scale and would end up a good bit faster a a larger n. You can also use a StringBuilder to take this approach, or String.repeat to efficiently create the lines.